src/EditorFeatures/VisualBasicTest/InlineMethod/VisualBasicInlineMethodTests.vb

```diff
@@ -0,0 +1,395 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+Imports Microsoft.CodeAnalysis.Editor.UnitTests.CodeActions
+Imports Microsoft.CodeAnalysis.VisualBasic.CodeRefactorings.InlineTemporary
+
+Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.InlineMethod
+
+    <Trait(Traits.Feature, Traits.Features.CodeActionsInlineMethod)>
+    Public Class VisualBasicInlineMethodTests
```

Comment:
i'm surprised by how much smaller this test file is than the C# version.  can you explain the disparity?

Reply:
I only test some core logic for VB. Right now there are 90-100 tests for C#. I can gradually increase the VB part maybe to 60-70

Reply:
VB test number is now 50.  There are still some scenarios in my mind need to test. Will continue adding that